src/traces/surface/attributes.js

```diff
@@ -95,9 +95,16 @@ module.exports = {
         valType: 'data_array',
         description: 'Sets the text elements associated with each z value.'
     },
-    zauto: colorscaleAttrs.zauto,
-    zmin: colorscaleAttrs.zmin,
-    zmax: colorscaleAttrs.zmax,
+    surfacecolor: {
+        valType: 'data_array',
+        description: [
```

Comment:
Can we have a more descriptive description? I'm still a little unclear on how this is used.